definitions/variable/energy/energy-secondary.yaml

```diff
@@ -56,10 +78,13 @@
     description: Total production of fossil natural gas
     unit: EJ/yr
 - Secondary Energy|Heat:
-    description: Total heat generation
+    description: Total centralized heat generation
     unit: EJ/yr
 - Secondary Energy|Heat|{Fuel}:
-    description: Total heat generation from {Fuel}
+    description: Total centralized heat generation from {Fuel}
+    unit: EJ/yr
+- Secondary Energy|Heat|Electricity|Heat Pumps:
+    description: Total centralized heat generation from heat pumps ("Electricity" is already covered in {Fuel})
```

Comment:
This seems to be a copy-paste error with {Fuel}

Reply:
sorry, I am failing to see the copy-paste error - could you clarify? @danielhuppmann

Reply:
There is a tag `{Fuel}` in the description but no corresponding tag in the variable name - and the whole second part of the description doesn’t make sense to me…

Reply:
I meant to explain why there is an explicit `Electricity|Heat Pumps` version of the `Secondary Energy|Heat|` variable, but no explicit `Electricity` version - that one is already included in the `{Fuel}` version of `Secondary Energy|Heat|`.

But I can easily remove this explanation if you find it self-explanatory

Reply:
I still don’t understand what you want to say. But in any case, please remove the curly brackets because are reserved for tags that are automatically replaced - and they can only be replaced if the same tag is present in the variable name itself.

Reply:
good to know. I just updated it accordingly